modules/savedb.py

```diff
@@ -0,0 +1,62 @@
+import time
+import json
+import MySQLdb
+import sys
+import argparse
+
+def saveToDatabase(database, user, password, links):
+        """
+        Connects to a MYSQL DB
+        Create MYSQL Table
+        Add link to the DB
+
+        Args:
+                data = data that is being stored in the database
+                user = username to login into MYSQL
+                password = password of MYSQL
+                link = URLs from the crawler
+        """
+        if database and user and password is None:
+                print("Wrong DB Credentials")
+                exit()
+
+        #Debug
+        #print("Database:", database, "\nuser:",user, "\npass:",password)
+        db = MySQLdb.connect(host="localhost",    # your host
```

Comment:
What if connection doesn't happen, error is not caught.

Reply:
well, it is true. What would the best fix? a `try expect` block with a `raise` in case of error?

Reply:
yes, ```try expect``` will do